### PR TITLE
[DISKPART] Update Polish (pl-PL) translation

### PR DESCRIPTION
**Pull request by pithwz**
diskpart update help in lang pl-pl

### CODE REVIEW

**Review by archeYR — APPROVED** [review]

**Review by Saibamen — APPROVED** [review]

File: base/system/diskpart/lang/pl-PL.rc [hunk 1]
```diff
@@ -32,10 +32,10 @@ BEGIN
     IDS_DETAIL_INFO_HIBER_FILE_DSK "Dysk pliku hibernacji"
     IDS_DETAIL_INFO_CRASH_DSK "Dysk zrzutu awarii systemu"
     IDS_DETAIL_INFO_CLST_DSK "Dysk klastrowany"
-    IDS_DETAIL_PARTITION_NUMBER "Partition %lu\n"
-    IDS_DETAIL_PARTITION_TYPE "Type          : %02x\n"
-    IDS_DETAIL_PARTITION_HIDDEN "Hidden        : %s\n"
-    IDS_DETAIL_PARTITION_ACTIVE "Active        : %s\n"
+    IDS_DETAIL_PARTITION_NUMBER "Partycja %lu\n"
+    IDS_DETAIL_PARTITION_TYPE "Typ          : %02x\n"
```
Comment (JoachimHenze) [inline base/system/diskpart/lang/pl-PL.rc:36]: a space too much before the colon

File: base/system/diskpart/lang/pl-PL.rc [hunk 2]
```diff
@@ -32,10 +32,10 @@ BEGIN
...
+    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
```
Comment (JoachimHenze) [inline base/system/diskpart/lang/pl-PL.rc:37]: one space too less before the colon

File: base/system/diskpart/lang/pl-PL.rc [hunk 3]
```diff
@@ -49,9 +49,9 @@ BEGIN
     IDS_LIST_PARTITION_LINE "  -------------  ----------------  -------  ------------\n"
     IDS_LIST_PARTITION_FORMAT "%c Partycja    %2lu   %-16s  %4I64u %-2s  %4I64u %-2s\n"
     IDS_LIST_PARTITION_NO_DISK "\nNie wybrano dysku do wyświetlenia partycji.\nWybierz dysk i spróbuj ponownie.\n\n"
-    IDS_LIST_VOLUME_HEAD "  Volume ###  Ltr  Label        FS     Type        Size     Status   Info\n"
-    IDS_LIST_VOLUME_LINE "  ----------  ---  -----------  -----  ----------  -------  -------  --------\n"
-    IDS_LIST_VOLUME_FORMAT "  Volume %3lu   %c   %-11s  %-5s  %10u  %4I64u %-2s\n"
+    IDS_LIST_VOLUME_HEAD "  Wolumin ###  Lit  Etykieta     FS     Typ         Rozmiar  Stan     Info\n"
+    IDS_LIST_VOLUME_LINE "  -----------  ---  -----------  -----  ----------  -------  -------  --------\n"
+    IDS_LIST_VOLUME_FORMAT "  Wolumin %3lu   %c   %-11s  %-5s  %10u  %4I64u %-2s\n"
```
Comment (JoachimHenze) [inline base/system/diskpart/lang/pl-PL.rc:54]: please attach a picture demonstrating that the indentation of this table does still look like intended after your changes.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: base/system/diskpart/lang/pl-PL.rc [hunk 4]
```diff
@@ -32,10 +32,10 @@ BEGIN
...
     IDS_DETAIL_INFO_CRASH_DSK "Dysk zrzutu awarii systemu"
     IDS_DETAIL_INFO_CLST_DSK "Dysk klastrowany"
-    IDS_DETAIL_PARTITION_NUMBER "Partition %lu\n"
-    IDS_DETAIL_PARTITION_TYPE "Type          : %02x\n"
-    IDS_DETAIL_PARTITION_HIDDEN "Hidden        : %s\n"
-    IDS_DETAIL_PARTITION_ACTIVE "Active        : %s\n"
+    IDS_DETAIL_PARTITION_NUMBER "Partycja %lu\n"
+    IDS_DETAIL_PARTITION_TYPE "Typ          : %02x\n"
+    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
+    IDS_DETAIL_PARTITION_ACTIVE "Aktywna        : %s\n"
```
Comment (JoachimHenze) [inline base/system/diskpart/lang/pl-PL.rc:38]: initial english:
    IDS_DETAIL_PARTITION_TYPE "Type          : %02x\n"
    IDS_DETAIL_PARTITION_HIDDEN "Hidden        : %s\n"
    IDS_DETAIL_PARTITION_ACTIVE "Active        : %s\n"

adding spaces before the opening " makes that easier to see for the human eye,
that your current:

    IDS_DETAIL_PARTITION_TYPE   "Typ          : %02x\n"
    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
    IDS_DETAIL_PARTITION_ACTIVE "Aktywna        : %s\n"
    IDS_DETAIL_PARTITION_OFFSET "Offset in Byte: %I64u\n"

should become:

    IDS_DETAIL_PARTITION_TYPE   "Typ           : %02x\n"
    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
    IDS_DETAIL_PARTITION_ACTIVE "Aktywna       : %s\n"
    IDS_DETAIL_PARTITION_OFFSET "Offset in Byte: %I64u\n"

Comment (pithwz) [reply base/system/diskpart/lang/pl-PL.rc:38]: it should/must be translated ;)...
You want list of volume in diskpart/cmd in ENGLISH ?😂🤣🤣🤣

File: base/system/diskpart/lang/pl-PL.rc (hunk 3 above)
Comment (pithwz) [reply base/system/diskpart/lang/pl-PL.rc:54]: You're funnny, you know?�
reactos_diskpart_volume
�

Comment (pithwz) [reply base/system/diskpart/lang/pl-PL.rc:54]: The "indentation ", should/must be bigger lok on winnt 5.2
winnt5x_diskpart_volume

File: base/system/diskpart/lang/pl-PL.rc (hunk 4 above)
Comment (pithwz) [reply base/system/diskpart/lang/pl-PL.rc:38]: fixed

**Review by binarymaster — APPROVED** [review]

**Comment by JoachimHenze** [comment]
I requested for a screenshot of the partition table before and after your change earlier, to see whether its layout is not broken.
What is with that? (The other parts that did lead to my reject once, were addressed correctly now)

**Comment by binarymaster** [comment]
> I requested for a screenshot of the partition table before and after your change earlier, to see whether its layout is not broken.

It was posted here: https://github.com/reactos/reactos/pull/4413#discussion_r841024903

File: base/system/diskpart/lang/pl-PL.rc (hunk 3 above)
Comment (JoachimHenze) [reply base/system/diskpart/lang/pl-PL.rc:54]: I see. Please file a fresh ticket about that!

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**